js1-2-8.js

```diff
@@ -0,0 +1,24 @@
+let firstNumber = 7,
+    secondNumber = 10;
+
+console.log("The value of my variable is: " + firstNumber);
+console.log(typeof(firstNumber));
+
+if(typeof(firstNumber) == typeof(secondNumber)){
+    console.log("SAME TYPE");
+}
+
+
+/* let firstNumber = 7,
+secondNumber = 10; */
+
+let firstString = "10",
+    first = 0;
+
```

Comment:
Ah, I think it's almost correct, I think you need to add two variables:

one for the `array` data-type and one of the `boolean` data type. Then compare those as well. Then the homework is approved!